### PR TITLE
Add a Request Membership action on organization page

### PR DESCRIPTION
**Pull request by maudetes**
Fix https://github.com/etalab/data.gouv.fr/issues/438.

Used a simple prompt. See the following:

image
image
image
image

For now, the pending status is not updated dynamically.

### CODE REVIEW

File: theme/js/components/organization/request-membership.vue [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+# Request Membership
+
+A simple request membership prompt.
+-->
+
+<template>
+    <li class="my-md">
+        <a @click.prevent="JoinOrga"
+      class="nav-link">{{ $t('Ask to join the organization as a producer') }}</a>
```
Comment (abulte) [inline theme/js/components/organization/request-membership.vue:15]: Was that the previous wording?

**Review by abulte — COMMENTED** [review]
Since we're going full vintage ™️ maybe reload the page after submit to update the request status? Might be violent though :godmode:

File: theme/js/components/organization/request-membership.vue (hunk 1 above)
Comment (maudetes) [reply theme/js/components/organization/request-membership.vue:15]: Nope, it used to be "join/rejoindre" only in legacy. I used the suggestion in https://github.com/etalab/data.gouv.fr/issues/438#issuecomment-915266108.

File: theme/js/components/organization/request-membership.vue [hunk 2]
```diff
@@ -36,12 +36,15 @@ data() {
...
-              alert(this.$t('Error while requesting membership'));
-              log.error(error);
-          });
+            alert(this.$t('A request has been sent to the administrators'));
+        })
+        .catch(error => {
+            alert(this.$t('Error while requesting membership'));
+            log.error(error);
+        })
+        .finally(()=> {
```
Comment (abulte) [inline theme/js/components/organization/request-membership.vue:45]: do we want to reload when there's an error?

Comment (maudetes) [reply theme/js/components/organization/request-membership.vue]: Nope, indeed!

File: theme/js/components/organization/request-membership.vue [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    return {
+        comment: '',
+    };
+},
+  methods: {
+    JoinOrga: function () {
+
+      this.$auth(this.$t("You must be connected to join an organization."));
+
+      this.comment = prompt(this.$t('You can add some details here for your membership request.'))
```
Comment (abulte) [inline theme/js/components/organization/request-membership.vue:35]:
```suggestion
      this.comment = prompt(this.$t('You can add some details here for your membership request.'));
```

**Review by abulte — APPROVED** [review]

**Pull request merged**